src/EditorFeatures/Core/Implementation/NavigationBar/NavigationBarController.cs

```diff
@@ -97,41 +132,17 @@ internal partial class NavigationBarController : ForegroundThreadAffinitizedObje
                 TaggerEventSources.OnWorkspaceRegistrationChanged(subjectBuffer));
```

Comment:
this is the line that notifies us when our buffer is attached to a workspace.  so no need for other components to tell us.